deploy/helm/values.yaml

```
@@ -7,8 +7,8 @@ replicaCount: 1
 # imagePullSecrets:
 
 image:
-  repository: quay.io/kubernetes_incubator/nfs-provisioner
-  tag: v2.3.0
+  repository: gcr.io/k8s-staging-sig-storage/nfs-provisioner
+  tag: canary
   # digest:
```

> Use stable version tags instead of canary

@kmova quay.io is already not working I'm considering to push it with digest instead:

```suggestion
  digest: sha256:cd73e83b0d493e452bd1ee63a6ed44b6c813ce77fc9473b1660c84966e24b934
```

---

@kvaps - just pushed v3.0.0 tag. Can we use this instead of the canary? 

As new PRs get merged the tag(canary) and digest will end up being different and am not sure about the implications.

---

Ok, for the example deployment files as well?

---

except for that one nitpick comment on tag name.. the new changes look good!